### ISSUE TITLE
talks - "more" link should be more descriptive

### ISSUE THREAD

**Issue by phette23** [code4lib/2020.code4lib.org#148]
"more..." doesn't tell users where it's headed, should add something involving the title of the talk itself e.g. `<a>more<span class=sr-only> about {{talk.title}}</span>...</a>`
